### ISSUE TITLE
URL shows up about:blank

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#12358]
### Description
Sometimes background tabs when focused shows up about:blank and reload page doesn't seem to bring back the original URL

### Steps to Reproduce
Unsure of steps

**Actual result:**
URL is replaced with about:blank

**Expected result:**
URL should be retained all the time

**Reproduces how often:**
Intermittent issue

### Brave Version

**about:brave info:**

Brave | 0.20.9
-- | --
V8 | 6.3.292.48
rev | b5aec10
Muon | 4.5.31
OS Release | 10.0.16299
Update Channel | Beta
OS Architecture | x64
OS Platform | Microsoft Windows
Node.js | 7.9.0
Brave Sync | v1.4.2
libchromiumcontent | 63.0.3239.108

**Reproducible on current live release:**
No

### Additional Information
cc: @LaurenWags @kjozwiak

**Comment by srirambv** [comment]
Easy repro steps
1. Open any site 
2. Ctrl+Click on a link
3. New opened link will show about:blank
4. Open https://news.google.com in a tab
5. Click on any link
6. If the newly opened link takes >2sec to load the page it retains about:blank
7. Ctrl+Click on any link and immediately switch to the tab, before the URL is loaded about:blank is shown
![12358-1](https://user-images.githubusercontent.com/17010094/34266092-f5ffc6ae-e66f-11e7-9d2c-88931e660751.gif)

**Comment by LaurenWags** [comment]
I can repro MacOS using 0.20.9

**Comment by luixxiul** [comment]
dupe https://github.com/brave/browser-laptop/issues/11796 ?

**Comment by srirambv** [comment]
Yup its a dupe. Did it happen with any link opened in a new tab as well @luixxiul  when you logged?

**Comment by luixxiul** [comment]
I think so.

**Comment by srirambv** [comment]
Ok Closing the issue as duplicate of #11796

**Comment by kjozwiak** [comment]
Even though this might be a dupe, I'm adding this into the `0.20.x` milestone as #11796 is in `0.20.x` as well. We'll retest this once the PR from #11796 lands and close it off if the issue has been resolved.
